### DISCUSSION TITLE
Prevent multiple instances of WiX Bundle from running simultaneously

### DISCUSSION THREAD

**Discussion by rwilm** [wixtoolset/issues#8730] [Questions]
Currently, I can run as many instances of my Bundle as I want. I would like to change this to allow only 1 instance of my WiX Bundle to run at a time. Any other instances of my bundle started while a bundle is already running should fail/not execute.

I am using the standard BA in a WiX v5 Bundle project. I did not find any documentation about expected behavior for this scenario or any built-in variables or element attributes that would help control this behavior. Please point me to anything I missed.

I would appreciate any suggestions on how to solve this. Thanks!

**Comment by robmen** [comment]
Duplicate of #8194
